x-pack/plugins/transform/server/types.ts

```diff
@@ -20,4 +21,9 @@ export interface Dependencies {
 export interface RouteDependencies {
   router: IRouter;
   license: License;
+  getStartServices: CoreSetup<PluginStartContract>['getStartServices'];
+}
+
+export interface PluginStartContract {
```

Comment:
i think this should be renamed `PluginStartDependencies` and `Dependencies` on line 15 should be renamed `PluginSetuptDependencies`

Reply:
Done in https://github.com/elastic/kibana/pull/131819/commits/36d639885f4609acb2982b083b9639e7a47b7956